lighthouse-core/report/styles/report.css

```diff
@@ -834,6 +834,43 @@ body {
   outline: 0;
 }
 
+.report-body__icon {
+  width: 24px;
+  height: 24px;
+  border: none;
+  cursor: pointer;
+  flex: 0 0 auto;
+  background-repeat: no-repeat;
+  background-position: center center;
+  background-size: contain;
+  background-color: transparent;
+  margin-left: 8px;
+}
+
+.runtime-config__icon {
+  width: 18px;
+  height: 18px;
+  background-size: contain;
+  transform: translate(0, 1px);
+  background-position: center center;
+  background-repeat: no-repeat;
+  background-size: contain;
+  background-color: transparent;
```

Comment:
I think I saw @brendankenny mention this somewhere else, and I've been thinking the same...

Can we start a new perfx.css stylesheet and dump all the perfx stuff in that? The template can inject it dynamically based the context. We want to limit stylesheet bloat for CLI/CRX/Devtools users.